bazel/repository_locations.bzl

```diff
@@ -863,6 +863,18 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-11-12",
         cpe = "N/A",
     ),
+    proxy_wasm_rust_sdk = dict(
+        project_name = "WebAssembly for Proxies (Rust SDK)",
+        project_desc = "WebAssembly for Proxies (Rust SDK)",
+        project_url = "https://github.com/proxy-wasm/proxy-wasm-rust-sdk",
+        version = "802cab2eabc1da106d8b8b7be2c7278b2bfadfbd",
```

Comment:
This commit came just after the 0.1.3 release - https://github.com/proxy-wasm/proxy-wasm-rust-sdk/releases/tag/v0.1.3. Is the plan to leverage latest commits or use releases for this new dependency?

Reply:
Releases, but the change for better Bazel integration was added after `v0.1.3` was tagged.